versions.json

```diff
@@ -63,8 +124,7 @@
         "3.5.1-scala2.12-java11-python3-ubuntu",
         "3.5.1-python3",
         "3.5.1",
-        "python3",
-        "latest"
+        "python3"
```

Comment:
Please remove spcial tag from 3.5.1: L98(python3-java17), L135 (r)、L127(python3)、L143(scala)

Otherwise, will generate wrong version in [1] step

[1] https://github.com/apache/spark-docker?tab=readme-ov-file#step-3-publish-spark-docker-official-images